test/change_health/response/claim/report/report_835_data_test.rb

```diff
@@ -68,7 +76,7 @@ class Report835DataTest < Minitest::Test
           total_actual_provider_payment_amount: '810.8',
           total_charge_amount: '226'
         )
-        expected_claim.keys.each do |attribute|
+        expected_claim.each_key do |attribute|
```

Comment:
Rubocop is getting to you! :)

Reply:
I turned rubocop on in vs code & it auto does this. Love it :D